db/migrate/20241112172605_add_index_to_form_email_matches_profile_logs.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToFormEmailMatchesProfileLogs < ActiveRecord::Migration[7.1]
+  disable_ddl_transaction!
+
+  def change
+    add_index :form_email_matches_profile_logs, [:user_uuid, :in_progress_form_id], unique: true, algorithm: :concurrently
```

Comment:
Please add if not exists: 
`add_index :form_email_matches_profile_logs, [:user_uuid, :in_progress_form_id], unique: true, algorithm: :concurrently, , if_not_exists: true`

Reply:
Updated 👍